app/android/src/uk/co/lutraconsulting/PositionTrackingService.java

```diff
@@ -143,22 +143,23 @@ public int onStartCommand( Intent intent, int flags, int startId ) {
         NotificationChannel serviceChannel = new NotificationChannel(
             CHANNEL_ID,
             "Foreground Service Channel",
-            NotificationManager.IMPORTANCE_HIGH
+            NotificationManager.IMPORTANCE_LOW
         );
         
         NotificationManager manager = getSystemService( NotificationManager.class );
         manager.createNotificationChannel( serviceChannel );
 
         // Build notification for position tracking
-        Intent notificationIntent = new Intent( this, PositionTrackingService.class );
+        Intent notificationIntent = new Intent( this, InputActivity.class );
 
         PendingIntent pendingIntent = PendingIntent.getActivity( this, 0, notificationIntent, PendingIntent.FLAG_IMMUTABLE );
 
         Notification notification = new Notification.Builder( this, CHANNEL_ID )
-            .setContentTitle( getText( R.string.notification_title ) )
-            .setContentText( getText( R.string.notification_message ) )
+            .setSmallIcon( R.drawable.ic_notification )
+            .setContentTitle( "Tracking in progress" )
+            .setColor( getResources().getColor( R.color.grassColor ) )
+            .setForegroundServiceBehavior( Notification.FOREGROUND_SERVICE_IMMEDIATE )
```

Comment:
Looks like setForegroundServiceBehavior() is Android >= 12 only, will need to make that optional

Reply:
Fixed